include/nbl/asset/utils/IIncludeHandler.h

```diff
@@ -17,18 +17,18 @@ class IIncludeHandler : public core::IReferenceCounted
 {
 	public:
 		_NBL_STATIC_INLINE_CONSTEXPR char* BUILTIN_PREFIX = "nbl/builtin/";
-		static bool isBuiltinPath(const std::string& _p)
+		static bool isBuiltinPath(const system::path& _p)
 		{
 			const size_t prefix_len = strlen(BUILTIN_PREFIX);
-			return _p.compare(0u, prefix_len, BUILTIN_PREFIX)==0;
+			return _p.string().compare(0u, prefix_len, BUILTIN_PREFIX) == 0;
```

Comment:
there's probably a method in `std::filesystem` to check if `_p` is a subpath of `path(BUILTIN_PREFIX)`

Reply:
Nah, i've looked for that kind of stuff before and it looks like this is the shortest way to do it. Alternatively, i can iterate through `path(BUILTIN_PREFIX)` and check whether all dirs are the same, but no one-line function found for that.